src/script_opt/ProfileFunc.cc

```diff
@@ -85,6 +85,12 @@ ProfileFunc::ProfileFunc(const Expr* e, bool _abs_rec_fields)
 		e->Traverse(this);
 	}
 
+ProfileFunc::ProfileFunc(const Stmt* s, bool _abs_rec_fields)
+	{
+	abs_rec_fields = _abs_rec_fields;
```

Comment:
This could use an initializer list instead of renaming the argument.

Reply:
Done.